wiki/Rules/hu.md

```diff
@@ -0,0 +1,79 @@
+3.  **Ha valaki némítást kap, vagy felfüggesztik a fiókját, hagyd őt békén, és ne kezdj róla beszélni.** Ha valaki felfüggesztést kap, az eset csakis rá, és az adminisztrátorokra vonatkozik - senki másra!
+4.  **Támogasd a tisztességes játékot, és ne adj figyelmet a csalóknak, vagy egyéb toxikus felhasználóknak.** Az effélék sajnos léteznek, és ha foglalkozol velük, az bátoríthatja őket a folytatásra, vagy másokat provokálhatnak, hogy viselkedjenek úgy, mint ők. Jelentsd őket a `!report` chat-paranccsal vagy használd a gombokat a profil oldalukon és lépj tovább.
+5.  **Tartsd a chat-et tisztán, és korhatár nélküli közösséghez illően, kerülve a felnőtt tartalmakat/témákat, ahol lehetséges.** Felnőtt tartalmak egyáltalán nem illőek egy korhatár nélküli közösségbe. Tegyél amit szeretnél privát üzenetekben (az ésszerűség határain belül), de hagyd a nyilvános csatornákat tisztán, mindenki érdekében.
+6.  **Legyél figyelmes, hogy mit osztasz meg másokkal, különös tekintettel a felkavaró/felzaklató tartalmakra vonatkozóan.** Ha azon tűnődsz, hogy helyénvaló-e az adott tartalmat linkelni, valószínűleg nem az. Kerüld a gyorsan villogó fényeket, a véres, másokat felkavaró, és egyéb felzaklató képeket.
+7.  **A linkek legyenek relevánsak és legális weboldalakra mutassanak.** Illegálisan közvetítő weboldalak (hivatalosan licenceltek, például Animelab vagy Crunchyroll rendben van) vagy bármiféle illegális/kalóz szoftverek, illetve ehhez hasonlóak nem elfogadottak, és nemkívánatosak a chat-en. Twitch linkek reklámozása más videojátékokra rendben van, de CSAK a `#videogames` csatornában.
+8. **Ne hirdess semmiféle külső chat szervert, Discord-ot, Teamspeak-et, illetve Skype szervereket/csoportokat beleértve.** Az osu! chat elsősorban az _osu!_-ról szól, és nem arról, hogy a saját szervereidet híreszteld. Tedd ezt privát üzenetekben, ha mindenképpen szükséges.
+9.  **Nyelvspecifikus csatornákban, a domináns nyelvet kell beszélned (Angol az #english-ben, Magyar a #hungarian-ben, stb.) ha beszélgetésben veszel részt.** Abban a csatornában, ahol a nyelv nincs kifejezetten előírva (vagy a csatorna nevéből nem lehet kikövetkeztetni, például Magyar a `#hungarian`-ben és Holland a `#dutch`-ban), ott mindig Angol. A `#lobby` és a `#help` csatornákban bármely nyelv megengedett, nem úgy, mint az összes többiben. *A [moderátori csapatnak](/wiki/People/Global_Moderation_Team) nem kell követnie ezt a szabályt, amikor a munkájukat végzik.*
+
+Fórum Szabályzat
+----------------
+
+1.  **Csak értelmes, érdekes tartalmakat/témákat posztolj.** A fórum elsősorban megvitatásokra van, szóval győződj meg arról, hogy elég energiát szántál arra, hogy ne rabold a többiek idejét.
+2.  **Tartsd a posztjaidat és a válaszaidat a korhatár nélküli közösséghez megfelelően.** Felnőtt tartalomnak semmi keresnivalója egy korhatár nélküli közösségben, ugyanakkor semmi sem állít meg ezeknek a megvitatása privát üzenetekben.
```

Comment:
You repeated "korhatár nélküli közösségben" twice which shouldn't be a problem but this is a short paragraph and it kinda hurts to read it more than once. Honestly, I have no idea how we could avoid this (I'm tired anyway), maybe we can't.